fog/view/enclave/api/src/lib.rs

```diff
@@ -235,8 +235,8 @@ pub enum Error {
     EnclaveNotInitialized,
     /// Cipher encryption failed: {0}
     Cipher(CipherError),
-    /// Fog View Shard query response collation error
-    QueryResponseCollation,
+    /// Fog View Shard query response collation error: {0}
+    QueryResponseCollation(String),
```

Comment:
this doesn't look to be used, suggest reverting and or removing since the error isn't used
```suggestion
```

Reply:
Done